json_templates/azure/c0-azure-base/ert-template.json

```diff
@@ -65,46 +65,325 @@
         "value": "mysql-proxy-lb.sys.{{pcf_ert_domain}}"
       },
       ".mysql_monitor.recipient_email": {
-        "value" : "c0-coreteam@pivotal.io"
+        "value": "c0-coreteam@pivotal.io"
```

Comment:
This should be a param for the pipeline, not defaulted to c0-coreteam@pivotal.io.